wemix/miner/miner.go

```diff
@@ -16,7 +16,7 @@ var (
 	AmPartnerFunc               func() bool
 	IsPartnerFunc               func(string) bool
 	AmHubFunc                   func(string) int
-	CalculateRewardsFunc        func(*big.Int, *big.Int, *big.Int, func(common.Address, *big.Int)) (*common.Address, []byte, error)
+	CalculateRewardsFunc        func(*params.ChainConfig, *big.Int, *big.Int, func(common.Address, *big.Int)) ([]byte, error)
 	VerifyRewardsFunc           func(*big.Int, string) error
```

Comment:
Suggestions  removing  unused functions or types that validate rewards

wemix/miner/miner.go
>var VerifyRewardsFunc func(*big.Int, string) error
func VerifyRewards(num *big.Int, rewards string) error

wemix/admin.go
>func verifyRewards(num *big.Int, rewards string) error 
func (ma *wemixAdmin) verifyRewards(r1, r2 []byte) -> has bugs